### ISSUE TITLE
Infow Window Option

### ISSUE THREAD

**Issue by l1ph** [xon52/x5-gmaps#31]
Hi is there a option to leave the InfowWindow Closed on the beginning?

Thx for your help

```
<gmaps-info-window ref="b" :options="optionsWindow"  >
          <ImmoObjectMap  v-bind:key="windowcontent.we" :immo="windowcontent"  />
        </gmaps-info-window>
```

i tried to close it when mounted but it is not working

this.$refs.b.open()

**Comment by xon52** [comment]
I may need to change the method name in the code to avoid confusion, but the `open()` method in the _InfoWindow_ component is a creation method (i.e. It shouldn't be called outside of creation). It calls the Google Maps function `open()` which creates and attaches the map instance the InfoWindow belongs to.

The Google Maps function `close()` simply removes the InfoWindow from the map instance it's attached to and any references to it as far as Google is concerned. I have called that in the `beforeDestroy()` lifecycle hook.

So in short, there is no `open()` `close()` function of _InfoWindow_, only create and destroy.

How you should do this is through Vue `v-if` or `v-show` (and setting them as false to begin with):
```html
<gmaps-info-window v-if="show" :options="options">
   <p>Example Text</p>
</gmaps-info-window>
```
